tests/test_poisson_fem_cube.py

```diff
@@ -147,7 +147,7 @@ def test_computeError(self):
 		from mozart.poisson.fem.cube import solve
 		from mozart.poisson.fem.cube import computeError
 		iter = 4
-		for degree in range(1,4):
+		for degree in range(1,2):
```

Comment:
Continuous Integration(Travis CI) provides only 10 minutes for testing. So we should reduce test time..